crates/polars-core/src/fmt.rs

```diff
@@ -313,7 +359,7 @@ impl Debug for Series {
                 "Series"
             ),
             DataType::Null => {
-                writeln!(f, "nullarray; shape: {}", self.len())
+                format_nullarray!(f, self.len(), "nullarray", self.name(), "Series")
```

Comment:
We should be able to reuse the `format_array` macro for the Null Datype. If you downcast the Series into a NullChunked   it should work.

Reply:
We should probably rename `nullarray` to `null` to be more consistent

Reply:
I couldn't see a direct way to downcast into NullChunked, so I added some logic in order to do that. Is this the correct way? It works, but not sure if this is the implementation you meant